dashboard/app/views/plc/user_course_enrollments/show.html.haml

```diff
@@ -0,0 +1,17 @@
+%p#notice= notice
+
+%p Hello #{@user_course_enrollment.user.name}, here is your progress for #{@user_course_enrollment.plc_course.name}
+
+-@user_course_enrollment.module_assignments.each do |module_enrollment|
+  %p
+    %strong #{module_enrollment.plc_learning_module.name}
+    - module_enrollment.task_assignments.each do |task_assignment|
+      %li #{task_assignment.plc_task.name} - #{task_assignment.status}
+
+-if @user_course_enrollment.module_assignments.count == 0
```

Comment:
using `empty?` here instead of `.count == 0` might generate a trivially more efficient query

````
irb(main):001:0> User.last.sections.count
  User Load (0.7ms)  SELECT  `users`.* FROM `users` WHERE `users`.`deleted_at` IS NULL  ORDER BY `users`.`id` DESC LIMIT 1
   (38.1ms)  SELECT COUNT(*) FROM `sections` WHERE `sections`.`user_id` = 14
=> 0
irb(main):002:0> User.last.sections.empty?
  User Load (0.7ms)  SELECT  `users`.* FROM `users` WHERE `users`.`deleted_at` IS NULL  ORDER BY `users`.`id` DESC LIMIT 1
  Section Exists (13.9ms)  SELECT  1 AS one FROM `sections` WHERE `sections`.`user_id` = 14 LIMIT 1
=> true
````